content/03.results.md

```diff
@@ -94,7 +94,7 @@ The ADT by cell counts matrix is normalized by first determining the ambient pro
 We skip normalization for cells with low-quality ADT expression, as indicated by `DropletUtils::cleanTagCounts()`.
 Although `scpca-nf` normalizes ADT counts, the workflow does not perform any dimensionality reduction of ADT data; only the RNA counts data are used as input for dimensionality reduction.
 The normalized ADT data are saved as an `altExp` within the processed `SingleCellExperiment` containing the normalized RNA data and is output to a file with the suffix `_processed.rds`.
-All files containing `SingleCellExperiment` objects and associated `altExp` objects are converted to `AnnData` objects and exported as separate RNA (`_rna.hdf5`) and ADT (`_adt.hdf5`) `AnnData` objects.
+All files containing `SingleCellExperiment` objects and associated `altExp` objects are converted to `AnnData` objects and exported as separate RNA (`_rna.h5ad`) and ADT (`_adt.h5ad`) `AnnData` objects.
```

Comment:
Can you file issues tracking the change to the figures in https://github.com/AlexsLemonade/scpca-paper-figures/ as well please? The little icons say `HDF5`.